docs/developers/contributing.md

```diff
@@ -112,6 +112,50 @@ Now you are all set to start developing with napari.
 
 If you wish to contribute documentation changes to napari, please read the [guide on contributing documentation](documentation/index.md). 
 
+(add-examples)=
+## Adding examples to the [Gallery](gallery)
+
+All of the examples in the [examples Gallery](gallery) are Python scripts that
+live under [the `examples` folder of the napari repository](https://github.com/napari/napari/tree/main/examples).
+Because of how this gallery is built, every such Python script needs a docstring
+containing at least a title (following the `.rst` format of docstrings in
+Python) and one or more _tags_. This ensures the example will be shown in the
+correct place in the table of contents for the gallery (see, for example, the
+[](tagoverview) page.)
+
+Here's an example of what that means. The example file
+[add_image.py](https://github.com/napari/napari/blob/main/examples/add_image.py)
+contains the following:
+
+```python
+"""
+Add image
+=========
+
+Display one image using the :func:`view_image` API.
+
+.. tags:: visualization-basic
+"""
+
+from skimage import data
+import napari
+
+# create the viewer with an image
+viewer = napari.view_image(data.astronaut(), rgb=True)
+
+if __name__ == '__main__':
+    napari.run()
+```
+
+The equals signs under "Add image" mark this as the example title. The text
+below is a short description of the example, and the tags are assigned by the
+use of the `.. tags::` directive. In this case, the only tag associated with
+this example is `visualization-basic`. Multiple tags can be separated with a
+comma.
+
+Note that the examples are the `.py` source files, and any outputs and images
+will be autogenerated when the documentation site is built.
```

Comment:
Do we also want to add a note that `napari.run()` *has* to be under the `__main__` guard?

Reply:
Oh, yes. That's a good point I think. For people who use mostly notebooks and interactive environments the `__main__` and `napari.run()` could be a bit of a mystery?

Reply:
Good point! I will explain that.